lib/i18n/authentication/recover/recover.i18n.dart

```diff
@@ -15,7 +15,8 @@ extension Localization on String {
               'Los tres Guardianes han aceptado su solicitud para recuperar su cuenta. \n Su cuenta se desbloqueará en 24 horas.',
           "Hours Left": 'Horas restantes',
           'Account recovered ': 'Cuenta recuperada',
-          "Oops, Something went wrong, try again later.": '"Ups! Algo salió mal, por favor trate mas tarde',
+          "There are no guardians for this account.": '"No hay guardianes para esta cuenta.',
```

Comment:
I see you know a little bit of spanish ;)
`'"No --> 'No`

Reply:
Wasnt me!